src/storage/mod.rs

```diff
@@ -1835,8 +1918,7 @@ mod tests {
                 1,
                 Options::default(),
                 expect_fail_callback(tx.clone(), 0, |e| match e {
-                    Error::Txn(txn::Error::Mvcc(mvcc::Error::Engine(EngineError::Request(..)))) => {
-                    }
+                    Error::Txn(txn::Error::Mvcc(mvcc::Error::Engine(EngineError::Other(..)))) => {}
```

Comment:
Why it is `Other` here?

Reply:
@MyonKeminta

Reply:
The logic of Prewrite has changed, now it checks lock first and then check write_cf, causing different error thrown here.

Reply:
I restored the logic of Prewrite and restored it else.